docs/data/date-pickers/base-concepts/base-concepts.md

```diff
@@ -96,6 +96,25 @@ A correct `aria-labelledby` value is assigned to the dialog component based on t
 Make sure to provide an `aria-labelledby` prop to `popper` and/or `mobilePaper` `slotProps` in case you are using Date and Time Pickers component with **hidden toolbar** and **without a label**.
 :::
 
+## Server interaction
+
+If the selected value is used to interact with server, you might want to avoid sending all the intermediate states.
+Especially if the user is setting the date thanks to arrow interaction.
+
+In such a case, the recommended UI is to add a button for validating the form.
+If for some reason, you need to send the data to the server without having the user pressing a validation button, you can debouce the `onChange` as follow.
+
+In the next demos, `<DateField />` is extended to add an `onAccept` props which is a debounced version of the `onChange`.
```

Comment:
```suggestion
In the following demo, `<DateField />` is extended adding an `onAccept` prop which is a debounced version of the `onChange`.
```

Reply:
@samuelsycamore could you give some hints on this topic?

I noticed that each time I want to express "the thing X has been done to get the result Y" I end up with the structure "X is <verb+ED> to <verb> Y" which Grammarly frequently corrects by "X is <verb+ED> <verb+ING> Y"

My problem is that: 
the first sentence feels like "X is done in order to get Y" (what I want to express)
the corrected sentence feels like "X is done the consequence being Y"

I don't know the correct expression, and if there is a better way to express this kind of idea.

Reply:
Grammarly can be helpful sometimes but it's not always trustworthy. It has odd idiosyncrasies and misunderstands what's being communicated. In this case the structure doesn't make sense. I think it stems from a larger problem, which is that the sentence itself uses passive voice, and Grammarly doesn't understand how the clauses are related. 

Here's how I would rephrase it to use active voice instead:

```suggestion
The following demo shows how to extend the Date Field component by adding an `onAccept` prop, which is a debounced version of `onChange`.
```